src/bucket/BucketList.cpp

```diff
@@ -241,12 +246,6 @@ BucketList::levelHalf(uint32_t level)
     return levelSize(level) >> 1;
 }
 
-uint32_t
```

Comment:
looks like the move/rename isn't relevant anymore? would it be possible to revert it, as less changes are easier to review